packages/api-docs-builder/buildApiUtils.ts

```diff
@@ -689,13 +689,23 @@ export const getStaticPaths = () => {
 ${staticProps}
       `;
 
```

Comment:
I'm reverting the changes to this file in #37941. It doesn't work per the last commit of #37931. I think that the markdown should match exactly the URL (/base-ui/all-components/) to make reasoning with the path easier. In our case, moving the markdown to match is enough to fix the issue on its own.

Reply:
Oh, I see, `yarn docs:api` now generates files but the CI doesn't check that no new files arent committed, chance the CI didn't catch the regression in #37931.

Reply:
I'm cleaning my mess: #38039.